### ISSUE TITLE
Javascript syntax string highlighting

### ISSUE THREAD

**Issue by zicsus** [Microsoft/vscode#57768]
Issue Type: <b>Bug</b>

When you make a string in javascript and concatenate the string with another string and put the second string on new line the syntax highligting starts to treat the string as normal code.

VS Code version: Code 1.26.1 (493869ee8e8a846b0855873886fc79d480d342de, 2018-08-16T18:38:57.434Z)

**Comment by gmarino2048** [comment]
Similar issue, when concatenating a string on a separate line, the linter detects the rest of the file as a string and highlights it as orange. (Been working with string interpolation, maybe that has something to do with it).

Also, code correction does NOT detect this as an error, just a problem with the color coding function.

### Problem code:
```
var fileContent = `<!DOCTYPE html>\n\n<html>\n<head>\n` + 
        `${body('head').html()}\n` +
        `\n</head>\n\n<body>` +
        `${header('body').html()}\n` + 
        `${body('body').html()}\n` +
        `${footer('body').html()}\n` +
        `\n</body>\n</html>`;
```

### Image of the bug:
![vscode_bug](https://user-images.githubusercontent.com/23267637/45108863-e33d4480-b10b-11e8-892f-15ead3da5c28.PNG)

### Version Info:
```
Version: 1.26.1
Commit: 493869ee8e8a846b0855873886fc79d480d342de
Date: 2018-08-16T18:38:57.434Z
Electron: 2.0.5
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
```

Let me know if you need any more info.

**Comment by mjbvz** [comment]
@gmarino2048 That's already fixed in insiders and in VS code 1.27. Please open a new issue in the future

**Comment by mjbvz** [comment]
Closing as likely fixed in insiders
